src/index.html

```diff
@@ -11,7 +11,175 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      name="test-form"
+    >
+      <fieldset class="blocks">
+        <legend>Personal information</legend>
+
+        <label class="labeles surname">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="labeles">
+          Name:
+          <input
+            type="text"
+            name="name"
+            id="name"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="labeles">
+          How old are You?
+          <input
+            type="number"
+            name="yearsOld"
+            value="12"
+            min="1"
+            max="100"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="labeles">
+          Full date of birth:
+          <input
+            type="date"
+            name="birthData"
+          >
+        </label>
+
+        <label class="labeles">
+          I accept the term of the agreement:
+          <input
+            type="checkbox"
+            name="terms"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="blocks">
+        <legend>Registration</legend>
+
+        <label class="labeles email">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label class="labeles">
+          Password:
+          <input
+            type="password"
+            name="password"
+            required
+            minlength="5"
+            maxlength="15"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="blocks">
+        <legend>An interesting fact about you!</legend>
+
+        <label class="labeles loveCats">
+          Do you love cats?
+          <input
+            type="radio"
```

Comment:
The radio inputs for 'Do you love cats?' should have unique values to identify the selected option. For example, value="yes" and value="no".